PureBasicIDE/PureBasicIDE.pbp

```diff
@@ -1,20 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<project xmlns="http://www.purebasic.com/namespace" version="1.0" creator="PureBasic 6.03 beta 3 LTS (Windows - x64)">
```

Comment:
Please remove the changes in this file from the PR as it is not really needed and easily causes merge conflicts.